bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -31,71 +31,54 @@ rules:
   - get
   - list
   - update
-- apiGroups: [""]
-  resources:
-  - endpoints
-  verbs:
-  - create
-  - update
-  - patch
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: openshift-ovn-kubernetes-node
+{{ if .NETWORK_IDENTITY_ENABLE }}
+  # the name change is required to ensure that both bindings exist during upgrade to avoid disruptions
+  name: openshift-ovn-kubernetes-nodes-group
+{{ else }}
+  name: openshift-ovn-kubernetes-node-ic
+{{ end }}
   namespace: openshift-ovn-kubernetes
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: openshift-ovn-kubernetes-node
+  name: openshift-ovn-kubernetes-node-ic
 subjects:
+{{ if .NETWORK_IDENTITY_ENABLE }}
+- kind: Group
+  name: system:ovn-nodes
+  apiGroup: rbac.authorization.k8s.io
+{{ else }}
 - kind: ServiceAccount
   name: ovn-kubernetes-node
   namespace: openshift-ovn-kubernetes
+{{ end }}
```

Comment:
Should the service account not be defined if we are not using it in the role bindings?

Reply:
With the changes for the rbac-proxy to use the SA we have to keep rendering it.